zkg

```diff
@@ -2512,6 +2518,12 @@ def argparser():
         " the latest version tag, or if a package has none,"
         ' the default branch, like "main" or "master".',
     )
+    sub_parser.add_argument(
```

Comment:
We need a similar option for `test`.

It is pretty unfortunate that we need to add these flags for each and every command. Another option would be to add it once to `top_parser` so we'd have `zkg --ignore-git-deps install .` instead of `zkg install --ignore-git-deps .`, similarly to what is done right now for `--verbose`.

Reply:
> We need a similar option for test.

It's there. I added one for test, install, and bundle.

> It is pretty unfortunate that we need to add these flags for each and every command.

I agree, but it also felt weird adding it at the top level, but only having it be valid for 3 out of the 21 possible commands.